plugins/rest/rest_auth.go

```diff
@@ -408,6 +410,26 @@ func (ap *clientTLSAuthPlugin) NewClient(c Config) (*http.Client, error) {
 		return nil, err
 	}
 
+	if ap.CACert != "" {
+		caCert, err := ioutil.ReadFile(ap.CACert)
+		if err != nil {
+			return nil, err
+		}
+
+		var caCertPool *x509.CertPool
+		if ap.SystemCARequired {
+			caCertPool, err = x509.SystemCertPool()
+			if err != nil {
+				return nil, err
+			}
+		} else {
+			caCertPool = x509.NewCertPool()
+		}
+
+		caCertPool.AppendCertsFromPEM(caCert)
```

Comment:
https://golang.org/pkg/crypto/x509/#CertPool.AppendCertsFromPEM might be worth checking the return value? I imagine this could be hard to debug if something goes wrong.

Reply:
sure, that's a good catch. Will modify to check the return value.